bokeh/models/layouts.py

```diff
@@ -28,8 +28,24 @@ class Layout(Component):
     An optional height for the component (in pixels).
     """)
 
+    disabled = Bool(False, help="""
+    Whether the widget will be disabled when rendered. If ``True``,
+    the widget will be greyed-out, and not respond to UI events.
+    """)
+
+    # TODO: (mp) Not yet, because it breaks plotting/notebook examples.
+    # Rename to _repr_html_ if we decide to enable this by default.
+    def __repr_html__(self):
+        return notebook_div(self)
```

Comment:
> Let's just delete this?

This code is there to show that we can speak IPython's display protocol directly, just our examples have to be updated for this change. We can remove this though, but this PR is no place for that.